packages/block-library/src/comment-template/index.php

```diff
@@ -24,10 +24,37 @@ function render_block_core_comment_template( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$number = $block->context['queryPerPage'];
-
+	$per_page = $block->context['queryPerPage'];
 	// Get an array of comments for the current post.
-	$comments = get_approved_comments( $post_id, array( 'number' => $number ) );
+
+	$page = (int) get_query_var( 'cpage' );
+
+	$comment_args = array(
+		'number'  => $per_page,
+		'orderby' => 'comment_date_gmt',
+		'order'   => 'ASC',
+		'status'  => 'approve',
+		'post_id' => $post_id,
+		'offset'  => 0,
+	);
+
+	if ( $page ) {
```

Comment:
Under what circumstances would `get_query_var( 'cpage' );` be falsy here and why would that mean that we should return only the count of the comments? Asking out of ignorance here :)

Reply:
If you go to `http://localhost:8888/hello-world/` that is falsy. We get the comments count in order to calculate which should be the offset in order to show the last comments. 

For example, if you have 14 comments with 2 comments per page, show the 2 last ones. If you have  13 comments with 2 comments per page, show only one on the last page.

This, to be honest, I could not get it to work yet 😕

Reply:
Whoops, I missed your comment earlier. I think I get it now, thanks 😊